README.md

```diff
@@ -717,7 +717,7 @@ The IoT Agents provide two means to define those device groups:
 -   Static **Type Configuration**: configuring the `ngsi.types` property in the
```

Comment:
Please add a line to CHANGES_NEXT_RELEASE about this PR fixes. For instance:

```
Add: /iot/cgroups API endpoints (as equivalent to /iot/services) (#752)
```

Reply:
Fixed by https://github.com/telefonicaid/iotagent-node-lib/pull/780/commits/cd3818d4a7ac5de43f41130c134dbb3aee4a4fb8